### ISSUE TITLE
Replies look like notices/are greyed out

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#25788]
### Steps to reproduce

1. Replicate the screenshot

#### What did you expect?

Previous styling which represented the sender's replying text as high contrast, as though it was a regular message.

#### What happened instead?

![image](https://github.com/vector-im/element-web/assets/1190097/cb3c45c4-817b-4a87-a131-3d91ab38ce1f)

### Operating system

Windows 11

### Application version

Element Nightly version: 0.0.1-nightly.2023071301 Olm version: 3.2.14

### How did you install the app?

The Internet

### Homeserver

t2l.io

### Will you send logs?

No

**Comment by t3chguy** [comment]
![image](https://github.com/vector-im/element-web/assets/2403652/fe3497f9-7694-4d0e-9eb9-137283a78f9f)

This is a WCAG AA/AAA fail

**Comment by robintown** [comment]
I believe the trigger for this is the message being highlighted (doesn't have to be a reply)
